pom.xml

```diff
@@ -48,6 +51,37 @@
             <artifactId>jquery</artifactId>
             <version>1.7.2-1</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>junit</artifactId>
+            <version>1.14-SNAPSHOT</version>
+        </dependency>
+
+        <!-- test dependencies -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>2.0</version>
```

Comment:
:bug: you cannot use 2.x Pipeline plugins with a 1.609.x baseline. The last release compatible with this baseline was 1.14.2.